general/src/protocol/user.rs

```diff
@@ -39,7 +39,7 @@ pub struct User {
     pub ip_address: Annotated<IpAddr>,
 
     /// Username of the user.
-    #[metastructure(pii = "true", max_chars = "enumlike", skip_serialization = "empty")]
+    #[metastructure(pii = "false", max_chars = "enumlike", skip_serialization = "empty")]
```

Comment:
Has this been changed because the Sentry data scrubber does not touch username?

Reply:
yes